themes/stanford-style/lms/templates/footer.html

```diff
@@ -0,0 +1,37 @@
+## mako
```

Comment:
@ziafazal these templates were picked from existing stanford theme so that it may be overridden by operators.